apps/web/components/Embed.tsx

```diff
@@ -1133,12 +1133,7 @@ export const EmbedDialog = () => {
   const router = useRouter();
   const embedUrl: string = router.query.embedUrl as string;
   return (
-    <Dialog
-      name="embed"
-      clearQueryParamsOnClose={queryParamsForDialog}
-      onOpenChange={(open) => {
-        if (!open) window.resetEmbedStatus();
```

Comment:
`resetEmbedStatus` is not needed now as we haven't been using sessionStorage to detect embed mode. As sessionStorage was common for Embed Code Generator preview window and the app, it was to remove the side effect.